tests/operators/hive_operator.py

```diff
@@ -163,6 +163,13 @@ def test_hive_stats(self):
                 dag=self.dag)
             t.run(start_date=DEFAULT_DATE, end_date=DEFAULT_DATE, force=True)
 
+        def test_named_hive_partition_sensor(self):
```

Comment:
Testing could be a bit more comprehensive (e.g. make sure exception is thrown), confirming correct state/return value of poke after some calls etc.